src/components/Tricots.js

```diff
@@ -1,52 +1,62 @@
 import React, {Component} from "react";
 import {Header, List, Image} from "semantic-ui-react";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import of 'List'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1128)